### ISSUE TITLE
Feature: Add class to blockqoutes

### ISSUE THREAD

**Issue by HashandSalt** [getkirby/editor#28]
Consider Adding a class to blockqoutes. Great for art directed posts.

**Comment by nilshoerrmann** [comment]
I think classes and ids (for anchored links) are something that might be interesting for any of the block types. Is there a way to implement general settings to add classes and ids that can be enabled for all block types (custom ones as well)?

**Comment by texnixe** [comment]
Yes, very much in favor of being able to add classes to any block type.

**Comment by nilshoerrmann** [comment]
As mentioned in #109, it would be important to add some kind of marker/pointer to blocks with class settings because they will otherwise be invisible to the users (unless they check their frontend).
